cassandra/metadata.py

```diff
@@ -168,10 +168,14 @@ def _rebuild_all(self, parser):
         current_keyspaces = set()
         for keyspace_meta in parser.get_all_keyspaces():
             current_keyspaces.add(keyspace_meta.name)
-            old_keyspace_meta = self.keyspaces.get(keyspace_meta.name, None)
+            old_keyspace_meta: KeyspaceMetadata = self.keyspaces.get(keyspace_meta.name, None)
             self.keyspaces[keyspace_meta.name] = keyspace_meta
             if old_keyspace_meta:
                 self._keyspace_updated(keyspace_meta.name)
+                for table_name, old_table_meta in old_keyspace_meta.tables.items():
+                    new_table_meta: TableMetadata = keyspace_meta.tables.get(table_name, None)
+                    if new_table_meta is None:
+                        self._table_removed(keyspace_meta.name, table_name)
```

Comment:
🔧 `old_table_meta` is unused. Why not `for table_name in old_keyspace.tables.keys()`?

Reply:
removed